src/commands/subdomain.rs

```diff
@@ -58,13 +70,34 @@ fn subdomain_addr(account_id: &str) -> String {
     )
 }
 
-pub fn subdomain(name: &str, user: &GlobalUser, target: &Target) -> Result<(), failure::Error> {
+pub fn subdomain(
+    name: Option<&str>,
+    user: &GlobalUser,
+    target: &Target,
+) -> Result<(), failure::Error> {
     if target.account_id.is_empty() {
         failure::bail!(format!(
             "{} You must provide an account_id in your wrangler.toml before creating a subdomain!",
             emoji::WARN
         ))
     }
+    if name.is_none() {
+        return match Subdomain::get_as_option(&target.account_id, user) {
+            Ok(Some(subdomain)) => {
+                println!("{}.workers.dev", subdomain);
```

Comment:
Should this use `println`, `print`, or `message::info`? Unsure if it'd be useful for this to be piped to stdin – probably not?